internal/api/grpc/admin/sms.go

```diff
@@ -65,8 +65,36 @@ func (s *Server) UpdateSMSProviderTwilioToken(ctx context.Context, req *admin_pb
 	}, nil
 }
 
+func (s *Server) AddSMSProviderHTTP(ctx context.Context, req *admin_pb.AddSMSProviderHTTPRequest) (*admin_pb.AddSMSProviderHTTPResponse, error) {
+	id, result, err := s.command.AddSMSConfigHTTP(ctx, authz.GetInstance(ctx).InstanceID(), AddSMSConfigTHTTPToConfig(req))
+	if err != nil {
+		return nil, err
+	}
+	return &admin_pb.AddSMSProviderHTTPResponse{
+		Details: object.DomainToAddDetailsPb(result),
+		Id:      id,
+	}, nil
+}
+
+func (s *Server) UpdateSMSProviderHTTP(ctx context.Context, req *admin_pb.UpdateSMSProviderHTTPRequest) (*admin_pb.UpdateSMSProviderHTTPResponse, error) {
+	result, err := s.command.ChangeSMSConfigHTTP(ctx, authz.GetInstance(ctx).InstanceID(), req.Id, UpdateSMSConfigHTTPToConfig(req))
```

Comment:
since it's only a single string, why even create a struct to pass it
you anyway already pass the id separately